tests/h/presenters/annotation_html_test.py

```diff
@@ -1,130 +1,109 @@
 import datetime
-from unittest import mock
 
-import jinja2
 import pytest
+from jinja2 import Markup
 
 from h.presenters.annotation_html import AnnotationHTMLPresenter
 
 
 class TestAnnotationHTMLPresenter:
```

Comment:
There's a strange thing going on here where the `AnnotationHTMLPresenter` acts nothing like the other presenters. It's a composed object which masquerades as an "annotation" and provides all sorts of proxies to it's methods.

This is pretty horrible, and good luck working out which of these methods are used where. But it also introduces we weirdness where the presenter is referred to as "annotation" in all the code that uses it. So in two pieces of code you can have:

 * `annotation.document` - The document model of the annotation model
 * `annotation.document` - The document presenter of the annotation presenter
 
This lead to some very confusing stuff here because there are two "annotations" in this world. I've standardised on calling the presenter "presenter" and I think it's a lot clearer as a result.